docs/guide/advanced/composition.md

```diff
@@ -413,7 +413,6 @@ In this example, the definition of resources is separated from i18n custom block
 
 ### Shared locale messages for components
 
-In Leacy API mode, shared locale messages are used in components with the `sharedMessages` option.
```

Comment:
Ah, 
This message is a NOTE to users using legacy mode. So please don't delete it; keep it as a NOTE using Vitepress custom containers syntax. 🙏 
https://vitepress.dev/guide/markdown#custom-containers

Reply:
oh my.
my apologies. 
It was meant to be a correction on the spelling for `Legacy` instead of `Leacy`